### PR TITLE
[WIP] Gastric Pump Device

### PR DESCRIPTION
**Pull request by AlvCyktor**
# Document the changes in your pull request
WIP

- [x] Gastric Pump Sprites

- [ ] Pump/Bone setter design code

- [ ] Surgery code

# Spriting
gastric pump
guh1
guh2

# Wiki Documentation

uhh adds stuff I will edit this when this is done
# Changelog

:cl:  
rscadd: Stomach sucky 
/:cl:

### CODE REVIEW

File: code/modules/surgery/tools.dm [hunk 1]
```diff
@@ -25,13 +25,22 @@
...
 
+/obj/item/gastric_pump
+	name = "gastric pump"
+	desc = "a pump and a valve that connects directly to your stomach."
+	icon_state = "gastric_pump"
+	toolspeed = 0.85
+	materials = list(/datum/material/iron=3000, /datum/material/glass=1500)
+	flags_1 = CONDUCT_1
+	item_flags = SURGICAL_TOOL
+	tool_behaviour = TOOL_RETRACTOR
```
Comment (VaelophisNyx) [inline code/modules/surgery/tools.dm:42]: TOOL_BONESET or a new define in code/_defines/tools.dm for like, TOOL_STOMPUMP or something, would be best here imo

Comment (VaelophisNyx) [reply code/modules/surgery/tools.dm:42]: well rather if this is the basic one, a new define would probably be best

Comment (AlvCyktor) [reply code/modules/surgery/tools.dm:42]: yeah I forgot to do this oopsie :3

**Review by Ebin-Halcyon — CHANGES_REQUESTED** [review]
Right, so. I'm gonna give comments and pointers to the sprite.
Color palette, no bueno. Try to limit the color palette to 5-6 or less by using contrasting colors by hueshifting things.
image

Try to avoid minor color differences and gradienting that can barely be seen.
image

Here's my own take on it with a less bloated palette along with contrasting colors.
image

Along with a 1:1 image if you wanna study it.
180928411-90047308-34e3-4913-a8d1-f97612f56c92


If you have any other questions feel free to @ me or something.

**Comment by AlvCyktor** [comment]
> Right, so. I'm gonna give comments and pointers to the sprite. Color palette, no bueno. Try to limit the color palette to 5-6 or less by using contrasting colors by hueshifting things. image
> 
> Try to avoid minor color differences and gradienting that can barely be seen. image
> 
> Here's my own take on it with a less bloated palette along with contrasting colors. image
> 
> Along with a 1:1 image if you wanna study it. 180928411-90047308-34e3-4913-a8d1-f97612f56c92
> 
> If you have any other questions feel free to @ me or something.

thank you

**Pull request closed without merging**